src/commonMain/kotlin/exchange.dydx.abacus/state/manager/V4StateManagerAdaptor.kt

```diff
@@ -1128,13 +1151,26 @@ class V4StateManagerAdaptor(
     private fun trackOrderSubmit(
         uiClickTimeMs: Double,
         analyticsPayload: IMap<String, Any>?,
+        isTriggerOrder: Boolean,
         isCancel: Boolean = false
     ): Double {
         val submitTimeMs = Clock.System.now().toEpochMilliseconds().toDouble()
         val uiDelayTimeMs = submitTimeMs - uiClickTimeMs
```

Comment:
I missed this before, why are we converting these timestamps to Double? Shouldn't we be using Long?

Reply:
hmm unsure - this is just the existing pattern; cc @aforaleka who may have more context (I think you added these values for analytics?)